test/geometry/test_conversions.py

```diff
@@ -1584,6 +1602,11 @@ def test_everything(self, batch_size, device, dtype):
 
         assert_close(R, R2, rtol=1e-4, atol=1e-5)
         assert_close(t, t2, rtol=1e-4, atol=1e-5)
+
+    @pytest.mark.parametrize("batch_size", [5])
+    def test_gradcheck(self, batch_size, device, dtype):
```

Comment:
gradcheck is just for fp64, I have been removing the `dtype` for these tests to have fewer tests, and consecutively a faster tests suite
```suggestion
    def test_gradcheck(self, batch_size, device):
```